### PR TITLE
feat: add `alloy-core` prelude crate

### PR DESCRIPTION
**Pull request by DaniPopes**

### CODE REVIEW

File: crates/core/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+[package]
+name = "alloy-core"
```
Comment (mattsse) [inline crates/core/Cargo.toml:2]: should this just be `alloy`?

Comment (DaniPopes) [reply crates/core/Cargo.toml:2]: Other repo

File: crates/core/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+../../README.md
```
Comment (onbjerg) [inline crates/core/README.md:1]: i assume this is a symlink and github is just being weird

**Review by onbjerg — APPROVED** [review]
good as a start

**Review by mattsse — APPROVED** [review]

**Review by Evalir — APPROVED** [review]
sweet! should we have a prelude crate for alloy-rs/alloy too?

File: crates/core/README.md (hunk 2 above)
Comment (DaniPopes) [reply crates/core/README.md:1]: yeah idk why they're not marked

**Comment by DaniPopes** [comment]
Yes, next up

**Pull request merged**